guide/popular-topics/builders.md

```diff
@@ -0,0 +1,132 @@
+# Builders
+
+Discord.js provides a `builders` package which contains a variety of utilities you can use when writing your Discord bot.
+To install the package run `npm install @discordjs/builders` in your terminal.
+
+## Formatters
+
+Formatters are a set of utility functions which format input strings into the given format.
+
+### Basic Markdown
+
+The Formatters provide functions to format strings into all the different Markdown styles supported by Discord.
+
+```js
+const { bold, italic, strikethrough, spoiler, quote, blockQuote, underscore } = require('@discordjs/builders');
+const string = 'Hello!';
+
+const boldString = bold(string);
+const italicString = italic(string);
+const strikethroughString = strikethrough(string);
+const spoilerString = spoiler(string);
+const quoteString = quote(string);
+const blockquoteString = blockQuote(string);
+const underscoreString = underscore(string);
+```
+
+### Links
+
+There are also two methods to format hyperlinks. `hyperlink()` will format the URL into a masked markdown link and `hideLinkEmbed()` will wrap the URL in `<>`, preventing it from embedding.
+
+```js {2,4-5}
```

Comment:
```suggestion
```js
```
A few of these snippets don't really need highlights, since they're placed under a section with a sentence above explaining each one